http.go

```diff
@@ -17,11 +17,12 @@ type resultContainer struct {
 	Success  bool      `json:"success"`
 	Message  string    `json:"message"`
 	Comments []Comment `json:"comments,omitempty"`
+	Count    *int       `json:"count,omitempty"`
 }
 
 func (res *resultContainer) render(w http.ResponseWriter) {
 	if res == nil {
-		res = &resultContainer{false, "Some internal error occurred", nil}
+		res = &resultContainer{false, "Some internal error occurred", nil, nil}
```

Comment:
With Count as a value instead of a pointer the initializer can be left unchanged. Count will assume the reasonable default of 0